apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -192,10 +199,12 @@ initializeIcons('https://my.cdn.com/path/to/icons/');`
               </CodeBlock>
             </li>
             <li>
-              <p>Reference core Fabric styles:</p>
+              <p>Reference core Fabric styles. <code>ms-Fabric</code> sets the font-family for all Fabric typography classes used in the element it's added to.</p>
```

Comment:
This is *awesome*. Love this change--thank you! One small suggestion would be to make this line a little more instructive so that it's very clear developers *should* add `ms-Fabric` to an element. That could look something like this:

> Add the `ms-Fabric` class to a containing element, such as `<body>`, to set the font-family for all Fabric typography classes used within that element.